archive/oldchangelog.html

```diff
@@ -3151,7 +3151,7 @@
 	  synchronized to dependent glyphs.
 	<LI>
 	  Enforce the "Use My Metrics" bit of a reference.<BR>
-	  In TrueType, if this bit is set on a reference then then width of the containing
+	  In TrueType, if this bit is set on a reference then width of the containing
```

Comment:
Yes, this should have been "then the", but really, skip this file at least. And again, blunt application of tools is not indicated where human languages are employed.